### DISCUSSION TITLE
Picasa-like performance on older hardware.

### DISCUSSION THREAD

**Discussion by ThomasLeigh** [d2phap/ImageGlass#1095] [General]
When it comes to image viewing for years now I stick to Google Picasa, because **this is the only app with which I can quickly and smoothly view photos** (it shows double-clicked images **instantaneously** while in case of ImageGlass I need to wait a tangible while) - either fit-to-screen or full-sized. The reason I mention Picasa is that **I much like the concept of ImageGlass** (I find it holds much promise) and **I would love to be able to benefit from it.** Unfortunately I use an old PC (Windows 7 32-bit, 2GB RAM), but since **Picasa is somehow able to run smoothly even on such a setup** I’ve thought that maybe You could figure out how to make Your app similarly smooth **even on older hardware**. I realize that this might not be of Your interest. If so, however, it would be great for many more people to become able to benefit from Your great app.

**Comment by ThomasLeigh** [comment]
It’s Picasa 3.9.140 compilation 248, You can download it [from here](https://trailofthelight.xyz/file_download/9/picasa39-setup.exe).

**Reply by d2phap** [reply]
Thanks @ThomasLeigh 
IG uses WinForms framework which has limited in performance (is not GPU-accelerated).
I have been working on the new version for better performance and experience.

**Reply by ThomasLeigh** [reply]
Thank You for Your answer, I’m very, very happy and I can’t wait for a new release! :) **Thank You so much!**
